src/GrpcProtos/capture.proto

```diff
@@ -433,6 +434,14 @@ message FullCallstackSample {
   uint64 timestamp_ns = 4;
 }
 
+message TracepointCallstack {
+  // This is an internal proto. This will be used to merge callstacks
```

Comment:
Nit: `internal` might not be clear to the reader. Could you maybe explain that this proto is not supposed to leave the service and the client doesn't need to handle it.

Reply:
Done.